### PR TITLE
Fix typos in German print dialogue

### LINKED ISSUE #895: BUG: Minor typos in German "PDF drucken ..." dialogue
**Bug description:**
The German dialogue "PDF drucken ..." has minor typos as you can see in the attached screenshot. 

Bildschirmfoto vom 2020-04-20 19-11-22

To name them:
- After `…keine Dokumente drucken.` the `\n` of the string `QString msg = tr("Unfortunately, this version of %1 is unable to print PDF documents due to various technical reasons.\n").arg(QString::fromLatin1(TEXWORKS_NAME));` in line 1026 of the file [texworks/src/PDFDocumentWindow.cpp](https://github.com/TeXworks/texworks/blob/641fd0c7bc8c76978b8504ae166544db51b67aa4/src/PDFDocumentWindow.cpp) is missing.
- I would start the German as the english sentence in the bracket with a big letter as in `Bitte schließen Sie…`.

**Steps to reproduce the problem:**

1. Try to print a file with the flatpak package

**General information:**
_TeXworks version_: 0.6.5
_TeXworks obtained from_: Flathub
_Operating system_: Ubuntu 18.04

### PR DESCRIPTION
**Pull request by jakobjakobson13**
Minor fixes in the translation and a grammatical change in the original English text string.

Fixes #895

### CODE REVIEW

File: trans/TeXworks_de.ts [hunk 1]
```diff
@@ -362,8 +362,7 @@ Wollen Sie fortfahren?</translation>
     </message>
     <message>
         <location filename="../src/PDFDocumentWindow.cpp" line="1026"/>
-        <source>Unfortunately, this version of %1 is unable to print PDF documents due to various technical reasons.
-</source>
+        <source>Unfortunately, this version of %1 is unable to print PDF documents due to various technical reasons.</source>
         <translation>Diese Version von %1 kann aufgrund verschiedener technischer Gründe leider keine PDF Dokumente drucken.</translation>
```
Comment (stloeffler) [inline trans/TeXworks_de.ts:366]: I think the newline at the end of the source text should persist (it is in the source code after all!). It should, however, also be included in the translation (to ensure the correct white spacing in the dialog).

**Comment by stloeffler** [comment]
Thanks for your contribution. Please see code review for comments.

**Comment by jakobjakobson13** [comment]
The "Unfortunately" line is now in agreement with the other language files.

**Comment by stloeffler** [comment]
Thanks for looking into this. However, the "unfortunately" entry should have a newline character at then end - as in other language files (I randomly checked [fr](https://github.com/TeXworks/texworks/blob/master/trans/TeXworks_fr.ts#L318-L319) and [nl](https://github.com/TeXworks/texworks/blob/master/trans/TeXworks_nl.ts#L714-L715); note that the `</source>` is on a separate line of its own). BTW, this is also what running `lupdate` produces. Note that it's an explicit "\n" in the source code, but a "real" new line in the translation files.

**Comment by jakobjakobson13** [comment]
Okay, I get your arguments. However, in the screenshot of the print window I posted in #895 the text does not show a new line after "…drucken" ("…print."). The same applies for the french version, in the dutch versions its harder to tell, there it might look like a new line due to the line break.

Bildschirmfoto vom 2020-04-29 11-02-04

**Comment by stloeffler** [comment]
Sorry, I was looking at the source texts in those languages. Both definitely have the newline in the source message (as linked above). So the source message should definitly include the linebreak (otherwise Qt might not even pick up the translation at all as it looks for a message with a newline character at the end).

You are correct that the French translation is missing the line break in the translated message as well. The Dutch version does (correctly) have the newline/linebreak at the and of the translated message.

The intended formatting is with a (forced) line break as is evident from the English version.

**Comment by jakobjakobson13** [comment]
So how should it be fixed then?

**Comment by stloeffler** [comment]
The source text should be as it was (with a line break at the end - that's how it is referenced in the code), but the translation should be adapted to include that same line break at the end as well.

**Comment by jakobjakobson13** [comment]
So this would need to be done in each single translation file?

**Comment by stloeffler** [comment]
Yes - all translation files that are affected by this, anyway. But this is beyond the scope of this issue (according to its current title - which just refers to the German translation). In fact, I am hesitant to modify languages that I don't know/understand...

FWIW, changing the code (e.g. to not include the line break in the translatable string) would mean that one also has to change all translation files (to make them match the new code) plus change all the translations that already include the line break right now in order to remove it.

**Comment by jakobjakobson13** [comment]
As the fixing of the white spacing is a more general problem, I'm withdrawing this pull request.

**Comment by stloeffler** [comment]
I agree that it is a "more general" problem in the sense that it affects multiple translations. But that just means that different translators have to fix it for "their" translation. You can still propose a pull request for fixing it for the German translation.

**Pull request closed without merging**